zhaquirks/legrand/wire_pilot.py

```diff
@@ -0,0 +1,130 @@
+"""Module for Legrand Cable Outlet with pilot wire functionality."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import add_to_registry_v2
+import zigpy.types as t
+from zigpy.zcl import ClusterType
+from zigpy.zcl.foundation import (
+    BaseAttributeDefs,
+    BaseCommandDefs,
+    Direction,
+    ZCLAttributeDef,
+    ZCLCommandDef,
+)
+
+from zhaquirks.legrand import LEGRAND, MANUFACTURER_SPECIFIC_CLUSTER_ID
+
+MANUFACTURER_SPECIFIC_CLUSTER_ID_2 = 0xFC40  # 64576
+
+
+WIRE_PILOT_MODE_ATTR = 0x4000
+
+LEGRAND_HEAT_MODE_ATTR = 0x00
+
+
+class LegrandCluster(CustomCluster):
+    """LegrandCluster."""
+
+    cluster_id = MANUFACTURER_SPECIFIC_CLUSTER_ID
+    name = "LegrandCluster"
+    ep_attribute = "legrand_cluster"
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        device_mode = ZCLAttributeDef(
+            id=0x0000,
+            type=t.data16,
+            is_manufacturer_specific=True,
+        )
+        led_dark = ZCLAttributeDef(
```

Comment:
What does this do and should it have some sort of config entity in the quirk?

Reply:
It was here before so I didn't touched it. I will look in details and I may provide a config switch for those attributes if it makes sense in another PR.